gather/templates/gather_event_view.html

```diff
@@ -1,4 +1,4 @@
-{% extends "base.html" %}
+s% extends "base.html" %}
```

Comment:
```suggestion
{% extends "base.html" %}
```

Reply:
Yikes thanks! I guess I missed "Cmd" when hitting "Cmd-S". 😬

I've been thinking we should write some tests which just render each URL route in the whole app and check a 200 status is returned. It'd be relatively low effort, it'd catch these sorts of bugs, and would also serve as stubs for better tests...

Reply:
Sounds like a plan.